os-3.10/scripts/provision.sh

```diff
@@ -109,3 +109,22 @@ ansible-playbook -i $inventory_file $openshift_ansible/playbooks/deploy_cluster.
 /usr/bin/oc create identity allow_all_auth:admin
 /usr/bin/oc create useridentitymapping allow_all_auth:admin admin
 /usr/bin/oc adm policy add-cluster-role-to-user cluster-admin admin
+
+# Create local-volume directories
+mkdir /var/local/kubevirt-storage
+mkdir /var/local/kubevirt-storage/local-volume
+mkdir /mnt/local-storage
+mkdir /mnt/local-storage/local/
+for i in {1..10}
+do
+  mkdir /var/local/kubevirt-storage/local-volume/disk${i}
```

Comment:
so actually you could condense L#s 114-117 down into this for loop using -p I think?

Reply:
I believe so